### PR TITLE
feat(sidebar): #MC-203 add sync button to external calendars

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
sync icon is next to checkbox for external events in sidebar.

## Checklist tests
go to calendar module, check that the icon appears. if you click on it you should get a timeout for the following route in the network:
http://localdev:8090/calendar/[[yourCalendar]]/ical?url=[[yourUrl]]

## Issue ticket number and link
#MC-203 https://jira.support-ent.fr/browse/MC-203

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/sass/global/_calendar-specifics.scss [hunk 1]
```diff
@@ -1,3 +1,14 @@
+@font-face {
```
Comment (alicedraillard) [inline src/main/resources/public/sass/global/_calendar-specifics.scss:1]: https://confluence.support-ent.fr/pages/viewpage.action?pageId=91570716

File: gulpfile.js [hunk 2]
```diff
@@ -22,7 +22,12 @@ var bundle = gulp.src('./node_modules/entcore/bundle/*')
 return merge(html, bundle);
 });
 
-gulp.task('webpack', ['copy-files'], () => {
+gulp.task('copy-mdi-font', ['copy-files'], function () {
```
Comment (alicedraillard) [inline gulpfile.js:25]: https://confluence.support-ent.fr/pages/viewpage.action?pageId=91570716

File: package.json [hunk 3]
```diff
@@ -49,6 +49,7 @@
     "gulp-append-prepend": "^1.0.9",
     "jest": "^24.7.1",
     "jest-environment-node": "^23.0.0",
-    "ts-jest": "^22.4.2"
+    "ts-jest": "^22.4.2",
+    "@mdi/font": "^4.5.95"
```
Comment (alicedraillard) [inline package.json:53]: https://confluence.support-ent.fr/pages/viewpage.action?pageId=91570716

File: src/main/resources/public/ts/directives/external-calendar-addition/external-calendar-addition.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ interface IViewModel {
 
     openExternalCalendarForm(): void;
     closeExternalCalendarForm(): void;
-    createExternalCalendar(calendar: Calendar): void;
+    createExternalCalendar(calendar: Calendar): Promise<void>;
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/external-calendar-addition/external-calendar-addition.ts:19]: fic methode typing

File: package.json (hunk 3 above)
Comment (JinDouang) [inline package.json:53]: A voir si tu peux pour la version 6 si tu veux

**Review by JinDouang — APPROVED** [review]

**Review by FlorentMr — APPROVED** [review]

**Pull request merged**